api/overview/azure/latest/messaging.eventgrid-readme.md

```diff
@@ -379,7 +380,7 @@ View more https://github.com/Azure/azure-sdk-for-net/blob/Azure.Messaging.EventG
 
 This project welcomes contributions and suggestions.
 Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution.
-For details, visit <https://cla.microsoft.com.>
+For details, visit [Contributor License Agreements](https://opensource.microsoft.com/cla/).
```

Comment:
This will be overwritten on the next package publish. Ensure that content in this package's README.md file is updated. 

I think that file is here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid/README.md